content/benefits/benefits-documents/release-notes/2024-05-22.md

```diff
@@ -0,0 +1,11 @@
+This release includes 2 new changes to the Benefits Documents API: 
+
+In order to enable dependents of deceased Veterans to upload and search for benefits related documents, we have added a new participantId parameter to the request objects for the following endpoints:
```

Comment:
does this return the documents themselves or a list of document names?

Reply:
This returns a list of document names, and applies only to surviving dependents. We received guidance from the Office of Mission Support (OMS) that only a veteran or their designated representative should have access to the contents of their eFolder while the veteran is alive.